cli/main.go

```diff
@@ -55,7 +55,7 @@ func Main() {
 func start(_ context.Context, cmd *cli.Command) error {
 	// Only print the version if the flag is set
 	if cmd.IsSet(versionFlag.Name) {
-		log.Infof("mev-boost %s\n", config.Version)
+		fmt.Printf("mev-boost %s\n", config.Version) //nolint
```

Comment:
probably better to do the `setupLogging` step first, which sets it to stdout, and keep using `log.Infof`.

Reply:
I think using `fmt.Printf` here is the right thing actually. Otherwise, the output is difficult to parse:

```
$ ./mev-boost --version
INFO[2025-01-31T08:45:58.966-06:00] starting mev-boost                            version=3c51355-dev
INFO[2025-01-31T08:45:58.966-06:00] mev-boost 3c51355-dev                         version=3c51355-dev
```

And this would break EthPillar's regex here:

```
MB=$(if [[ -f /etc/systemd/system/mevboost.service ]]; then printf "Mev-boost: $(mev-boost --version | sed 's/.*\s\([0-9]*\.[0-9]*\).*/\1/')"; else printf "Mev-boost: Not Installed"; fi)
```

Reply:
I've updated the PR to do this instead:

```go
fmt.Fprintf(cmd.Writer, "mev-boost %s\n", config.Version)
```

I believe this is most appropriate for CLI apps. Doesn't require `//nolint` either.